### PR TITLE
Pull out some constants.

### PR DESCRIPTION
**Pull request by andrew-moore-octo**
This PR extracts some important strings out to constants in `MessageExchangeStream`.

### CODE REVIEW

File: source/Halibut/Transport/Protocol/MessageExchangeStream.cs [hunk 1]
```diff
@@ -143,10 +149,7 @@ async Task<string> ReadLineAsync()
 
         public void IdentifyAsSubscriber(string subscriptionId)
         {
-            streamWriter.Write("MX-SUBSCRIBER ");
-            streamWriter.Write(currentVersion);
-            streamWriter.Write(" ");
-            streamWriter.Write(subscriptionId);
+            streamWriter.Write($"{MxSubscriber} {currentVersion} {subscriptionId}");
```
Comment (matt-richardson) [inline source/Halibut/Transport/Protocol/MessageExchangeStream.cs:152]: I wonder if we should make this a `WriteLine` instead? (and drop the `streamWriter.WriteLine();` on the next line)

File: source/Halibut/Transport/Protocol/MessageExchangeStream.cs [hunk 2]
```diff
@@ -156,8 +159,7 @@ public void IdentifyAsSubscriber(string subscriptionId)
 
         public void IdentifyAsServer()
         {
-            streamWriter.Write("MX-SERVER ");
-            streamWriter.Write(currentVersion.ToString());
+            streamWriter.Write($"{MxServer} {currentVersion}");
```
Comment (matt-richardson) [inline source/Halibut/Transport/Protocol/MessageExchangeStream.cs:162]: I wonder if we should make this a WriteLine instead?

**Review by matt-richardson — APPROVED** [review]
Nice one. A couple of minor suggestions.

File: source/Halibut/Transport/Protocol/MessageExchangeStream.cs (hunk 1 above)
Comment (andrew-moore-octo) [reply source/Halibut/Transport/Protocol/MessageExchangeStream.cs:152]: Agreed. The next PR targets this area so I'll do it as part of that PR.

File: source/Halibut/Transport/Protocol/MessageExchangeStream.cs (hunk 2 above)
Comment (andrew-moore-octo) [reply source/Halibut/Transport/Protocol/MessageExchangeStream.cs:162]: Will do this in the next PR.

**Pull request merged**